core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -3205,13 +3220,52 @@ class ReplicaManagerTest {
         threadNamePrefix: Option[String],
         quotaManager: ReplicationQuotaManager
       ): ReplicaFetcherManager = {
-        mockReplicaFetcherManager.getOrElse {
-          super.createReplicaFetcherManager(
-            metrics,
-            time,
-            threadNamePrefix,
-            quotaManager
-          )
+          mockReplicaFetcherManager.getOrElse {
```

Comment:
nit: did we have the indent correct here?

Reply:
Addressed